js/google_ima.js

```diff
@@ -2598,6 +2598,7 @@ require("../html5-common/js/utils/utils.js");
      */
     this.setVolume = function(volume, muteState)
     {
+      volumeWhenMuted = volume;
```

Comment:
shouldn't this be saved when we are actually about to mute and not everytime we set the volume?

Reply:
The bug was because we were doing just that. We could rename the variable to savedVolume to avoid confusion.

Reply:
..except there is already a savedVolume in the GoogleIMA class. Might still be confusing. Will try and come up with something suitable

Reply:
Shoot. Wonder if we can consolidate them in a future release.